### ISSUE TITLE
Removal of singleSearch

### ISSUE THREAD

**Issue by markus659** [origo-map/origo-server#14]
I think we should work towards removing singleSearch, since the same thing can be achieved using the normal "search". Currently it is redundant and can be confusing when configuring the search.

**Comment by MattiasSp** [comment]
I agree, but perhaps it should be marked as deprecated somewhere and then removed in a future release? Perhaps we should name versions of Origo Server and start making releases, as we do with Origo?

**Comment by MattiasSp** [comment]
On second thought, the cost of keeping it around for backwards compatibility is negligible, so I'm closing this for now.
